### PR TITLE
Balance size of columns in decklist view

### PR DESCRIPTION
**Pull request by 7sempra**
Also move deck name onto its own line to avoid weird wrapping
issues.

### CODE REVIEW

File: decklist.html [hunk 1]
```diff
@@ -16,17 +16,22 @@
...
-    <div class="side"></div>
+<div class="columns">
+    <div class="player" id="p1">
+        <div class="player-name"></div>
+        <div class="deck-name"></div>
+        <div class="deck"></div>
+        <div class="side-title">Sideboard</div>
+        <div class="side"></div>
+    </div>
+    <div class="column-spacer"></div>
```
Comment (kprevas) [inline decklist.html]: 😕

Comment (7sempra) [inline decklist.html]: It's fiiiiiiine.

Comment (kprevas) [inline decklist.html]: just anime operator bro

Comment (7sempra) [inline decklist.html]: Ugh, I hate fancy CSS rules. Done tho.

**Pull request merged**